content/5_Plat/Range_Sweep.mdx

```diff
@@ -139,42 +167,71 @@ template <class T> struct Seg {  // comb(ID,b) = b
 	}
 };
 
+struct Point {
+	int x, y, i, type;
+	Point(int x, int y, int i, int type) : x(x), y(y), i(i), type(type) {}
+	bool operator<(const Point &p) {
+		if (x == p.x) return y < p.y;
+		return x < p.x;
+	}
+};
+
+const int MX = 1e5 + 5;
 Seg<int> S;
 int N, P;
 int ans[MX];
-vi distinct_y;
-
-int query_min(int ind) { return S.query(0, ind); }
-void ins(int ind, int val) { S.upd(ind, val); }
-int y_index(int y) { return lb(all(distinct_y), y) - begin(distinct_y); }
+vector<int> distinct_y;
+int y_index(int y) {
+	return lower_bound(begin(distinct_y), end(distinct_y), y) -
+	       begin(distinct_y);
+}
 
 int main() {
-	setIO("boards");
+	ios_base::sync_with_stdio(0);
+	cin.tie(0);
```

Comment:
is fastio needed for ac?

Reply:
bump

Reply:
removed
tho removing does make the code twice as slow (does ac tho)